examples/DynamicProperties-sample.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Aggregation.V1.xml">
+    <edmx:Include Namespace="Org.OData.Aggregation.V1" Alias="Aggregation" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/UI.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.UI.v1" Alias="UI" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Analytics.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Analytics.v1" Alias="Analytics" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="DynamicProperties.examples" Alias="self" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+
+      <EntityType Name="Sales">
+        <Key>
+          <PropertyRef Name="Order"/>
+        </Key>
+        <Property Name="Order" Type="Edm.String" Nullable="false"/>
+        <Property Name="Country" Type="Edm.String" Nullable="false"/>
+        <Property Name="NumberOfItems" Type="Edm.Int16" Nullable="false"/>
+        <Property Name="Amount" Type="Edm.Decimal"/>
+        <Annotation Term="Analytics.AggregatedProperty" Qualifier="sum">
+          <Record>
+            <PropertyValue Property="Name" String="Total"/>
+            <PropertyValue Property="AggregationMethod" String="sum"/>
+            <PropertyValue Property="AggregatableProperty" PropertyPath="Amount"/>
+          </Record>
+        </Annotation>
+        <Annotation Term="Analytics.AggregatedProperty" Qualifier="max">
+          <Record>
+            <PropertyValue Property="Name" String="Maximum"/>
+            <PropertyValue Property="AggregationMethod" String="max"/>
+            <PropertyValue Property="AggregatableProperty" PropertyPath="Amount"/>
+          </Record>
+        </Annotation>
+        <Annotation Term="UI.Chart">
+          <Record>
+            <PropertyValue Property="ChartType" EnumMember="UI.ChartType/Bubble"/>
+            <PropertyValue Property="Dimensions">
+              <Collection>
+                <PropertyPath>Country</PropertyPath>
+              </Collection>
+            </PropertyValue>
+            <PropertyValue Property="Measures">
+              <Collection>
+                <PropertyPath>NumberOfItems</PropertyPath>
+              </Collection>
+            </PropertyValue>
+            <PropertyValue Property="DynamicMeasures">
+              <Collection>
+                <AnnotationPath>@Analytics.AggregatedProperty#sum</AnnotationPath>
+                <AnnotationPath>/self.Container/me/SalesOrders@Aggregation.CustomAggregate#WeightedAverage</AnnotationPath>
```

Comment:
This absolute path is nested within an annotation on the entity type "Sales", pointing to something that is only valid for one special use of this entity type.

How would this work if there are other uses of entity type "Sales", e.g. a top-level entity set for "Sales" next to the "me" singleton?

Reply:
Good observation.

The absolute path refers to an annotation, which defines a custom aggregate on "a special usage of the `Sales` entity type". For other uses of the same entity type, the custom aggregate is not defined/not allowed.

On the other hand, the `UI.Chart` annotation, of which this absolute path is a property, is defined for _all_ usages of the `Sales` entity type. I conclude that, for usages other than `me/SalesOrders`, the chart has effectively only _one_ dynamic measure: the aggregated property defined by the other annotation path.

Reply:
Suggested wording:
> If the annotation referenced by an annotation path does not apply to the same collection of entities            as the `UI.Chart` annotation, the annotation path MUST be silently ignored.

Reply:
Oh no, please not yet another one of those "meta" annotations.

The suggested wording is misleading because UI.Chart only applies to types, never directly to collections.

Reply:
The definition of `UI.Chart` says "Visualization of multiple data points", this is a collection. What about that?
> If the annotation referenced by an annotation path does not apply to the same collection of entities as the one being visualized according to the `UI.Chart` annotation, the annotation path MUST be silently ignored.